app/_hub/kong-inc/correlation-id/index.md

```diff
@@ -128,6 +128,58 @@ form parameter      | description
 
 ## FAQ
 
-### Can I see my correlation ids in my Kong logs?
+### Can I see my correlation IDs in my Kong logs?
 
-The correlation id doesn't show up in the Nginx access or error logs. However, you can use this plugin along with one of the Logging plugins, or store the id on your backend.
+You can see your correlation ID in the Nginx access or error logs if you edit your Nginx logging parameters.
+
+To edit your Nginx parameters, do the following:
+
+1. Locate [{{site.base_gateway}}'s template files](/gateway/latest/reference/configuration/#custom-nginx-templates) and make a copy of `nginx_kong.lua`.
+1. Add a `log_format` section on the root level of the config file which includes the
+  `$sent_http_Kong_Request_ID` variable.
+   In the following example, we created a new log format named `customformat`.
+   It's a copy of the default `combined` log format, but the last line adds
+   `$sent_http_Kong_Request_ID`, preceded by the string `Kong-Request-ID=`.
+   Marking the variable this way is optional, and will make testing the feature easier.
+   Further customize the `log_format` by adding or removing
+   [variables](http://nginx.org/en/docs/http/ngx_http_log_module.html):
+
+   ```
+   log_format customformat '$remote_addr - $remote_user [$time_local] '
+                 '"$request" $status $body_bytes_sent  '
+                 '"$http_referer" "$http_user_agent" '
+                 'Kong-Request-ID="$sent_http_Kong_Request_ID"';
+   ```
+
+1. Use your custom log format for the proxy access log phase. This means locating the following line:
+   
+   ```
+   access_log ${PROXY_ACCESS_LOG};
+   ```
+
+     Modifying it by adding the `customformat` format that we just created:
+
+   ```
+   access_log ${{PROXY_ACCESS_LOG}} customformat;
```

Comment:
```suggestion
   access_log ${% raw %}{{PROXY_ACCESS_LOG}}{% endraw %} customformat;
```
Need these tags, otherwise jekyll tries to parse it as a variable.

Further note: is it accurate to use two curly braces in this example, or it is a typo? @kikito